include/pgduckdb/pgduckdb_utils.hpp

```diff
@@ -103,4 +104,10 @@ DuckDBFunctionGuard(FuncType duckdb_function, const char* function_name, FuncArg
 
 std::string CreateOrGetDirectoryPath(std::string directory_name);
 
+duckdb::unique_ptr<duckdb::QueryResult> DuckDBQueryOrThrow(duckdb::ClientContext &context, const std::string &query);
+
+duckdb::unique_ptr<duckdb::QueryResult> DuckDBQueryOrThrow(duckdb::Connection &connection, const std::string &query);
+
+duckdb::unique_ptr<duckdb::QueryResult> DuckDBQueryOrThrow(const std::string &query);
+
```

Comment:
I think it'd be good to have `const char*` versions of this too. Generally when doing calling such a function we'll be in a Postgres C context.

Reply:
Yeah I'm coming with a dedicated PR that rationalize this.

Reply:
I've actually made so that they always throw PG errors for now (which was the case already)